### PR TITLE
New Website v1.0

### PR DESCRIPTION
**Pull request by molnard**
zkSNACKs website remade with similar technology considerations as the wasabiwallet website.

### CODE REVIEW

**Review by jmacato — COMMENTED** [review]
It looks really great! Only have a couple of comments:

1: Sometimes the animated video background in the first part of the website can interfere with the zkSNACKs logo on certain screen sizes (mine is 1600x900), perhaps we could add either shadows or overlay to improve contrast? 
image

2. Would be great if we can find a vectorized version of the logos instead of bitmaps in the `As Seen On` part. 
image

**Review by ghost — CHANGES_REQUESTED** [review]
This is very difficult to read.

screenshot_20200512_175856

Change `Which we are proud of` to `We are proud of`.

Adam is no longer the CTO, David should be there too.

Maybe add that zksnacks is an open source company.

**Comment by yahiheb** [comment]
> Change Which we are proud of to We are proud of.

ACK.

**Comment by yahiheb** [comment]
Clicking on `Mission` or `WHICH WE ARE PROUD OF`, the header hides the title:

Capture

It should be like this:

Capture1

Also there should be some separation between `Mission` and `Wasabi` contents.

The font is a bit small.

File: wwwroot/index.html [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+		<div class="banner-section" id="about">
+			<video poster="./assets/bg2_video_poster.png" autoplay muted loop preload="auto">
+				<source src="assets/bg2.webm" type="video/webm" />
+			</video>
+
+			<div class="jumbotron jumbotron-fluid">
+				<div class="container">
+					<div class="mb-4"><img src="assets/zksnacks-logo-white.png" style="width: 425px"> </div>
+					<p class="lead">
+						Nowadays it is all about individuals’ digital privacy, governments and organisations are
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:70]:
```suggestion
						Nowadays it is all about individuals’ <strong>digital privacy</strong>, governments and organisations are
```

File: wwwroot/index.html [hunk 2]
```diff
@@ -0,0 +1,534 @@
...
+						making steps to protect privacy in the digital world. We believe that privacy is both
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:71]:
```suggestion
						making steps to <strong>protect privacy in the digital world</strong>. We believe that privacy is both
```

File: wwwroot/index.html [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+						a fundamental human right and business need that should be preserved at all times.
+						This is why we focus on your digital financial privacy.
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:73]:
```suggestion
						This is why <strong>we focus on your digital financial privacy</strong>.
```

File: wwwroot/index.html [hunk 4]
```diff
@@ -0,0 +1,534 @@
...
+		</div>
+		<div class="mission-section" id="mission">
+			<div class="container">
+				<div class="row topdiv">
+					<div class="col-12">
+						<div class="fadethisdiv">
+							<div class="card-body">
+								<h2 class="card-title underline-title mb-4">Mission</h2>
+								<p class="card-text text-center">
+									zkSNACKs was established to research the possibilities for Bitcoin fungibility
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:86]:
```suggestion
									zkSNACKs was established to research the possibilities for <strong>Bitcoin fungibility</strong>
```

File: wwwroot/index.html [hunk 5]
```diff
@@ -0,0 +1,534 @@
...
+									enhancements and provide different products ensuring the desired privacy.
+									<a class="bitcoin-anonymous" href="https://bitcoin.org/en/protect-your-privacy">Many
+										believes that Bitcoin is anonymous </a>
+									but in reality the Blockchain is transparent.
+								</p>
+								<p class="card-mission text-center">
+									Our mission is to equip you with powerful tools to secure your privacy.
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:93]:
```suggestion
									Our mission is to equip you with powerful tools to <strong>secure your privacy</strong>.
```

File: wwwroot/index.html [hunk 6]
```diff
@@ -0,0 +1,534 @@
...
+		</div>
+		<div class="product-section" id="product">
+			<div class="container">
+				<div class="row topdivasd">
+					<div class="col-12 fadethisdivasd">
+						<div class="product-container m-auto h-100 text-center">
+							<img class="m-auto" alt="Wasabi Wallet" src="assets/wasabi_wallet.png" style="width:235px">
+							<div class="card-body">
+								<p class="card-text">
+									zkSNACKs’ flagship product is a privacy oriented Bitcoin wallet, called the Wasabi
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:136]:
```suggestion
									zkSNACKs’ flagship product is a <strong>privacy oriented Bitcoin wallet</strong>, called the <a class="go-to" href="https://wasabiwallet.io">Wasabi
```

File: wwwroot/index.html [hunk 7]
```diff
@@ -0,0 +1,534 @@
...
+									Wallet. <br>
+									<a class="go-to" href="http://wasabiwallet.io">Go to website.</a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:138]:
```suggestion
```

File: wwwroot/index.html [hunk 8]
```diff
@@ -0,0 +1,534 @@
...
+		<div class="product-section" id="product">
+			<div class="container">
+				<div class="row topdivasd">
+					<div class="col-12 fadethisdivasd">
+						<div class="product-container m-auto h-100 text-center">
+							<img class="m-auto" alt="Wasabi Wallet" src="assets/wasabi_wallet.png" style="width:235px">
+							<div class="card-body">
+								<p class="card-text">
+									zkSNACKs’ flagship product is a privacy oriented Bitcoin wallet, called the Wasabi
+									Wallet. <br>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:137]:
```suggestion
									Wallet</a>. <br>
```

File: wwwroot/index.html [hunk 9]
```diff
@@ -0,0 +1,534 @@
...
+								<span class="highlight"> Wasabi 1.0 </span> released on the 10th anniversary of
+								Bitcoin's White Paper
+							</div>
+						</div>
+					</div>
+					<div class="col-12 col-md-6 col-lg-2 mb-3 pr-0 line2">
+						<div class="card development-card h-100 px-2 py-3">
+							<img class="svg-icon mb-0" alt="Lightning Network integration" src="assets/interface.svg">
+							<div class="card-body development-card-body">
+								Wasabi Wallet listed on <a class="bitcoin-anonymous" href="http://bitcoin.org">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:170]:
```suggestion
								Wasabi Wallet listed on <a class="bitcoin-anonymous" href="https://bitcoin.org/" target="_blank">
```

File: wwwroot/index.html [hunk 10]
```diff
@@ -0,0 +1,534 @@
...
+
+		<div class="logo-section">
+			<div class="container">
+				<div class="row">
+					<div class="col-12">
+						<h2 class="text-center mb-5 card-title underline-logos-title"> As Seen On</h2>
+					</div>
+					<div class="col-lg-12 col-md-8 col-12 m-auto mt-3">
+						<div class="customer-logos">
+							<a class="slide" href="https://www.coindesk.com/privacy-bitcoin-wallets/" target="_blank">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:220]:
```suggestion
							<a class="slide" href="https://www.coindesk.com/privacy-bitcoin-wallets/" target="_blank" rel="external nofollow">
```

File: wwwroot/index.html [hunk 11]
```diff
@@ -0,0 +1,534 @@
...
+								<img src="assets/client-logos/CoinDesk_logo.png" alt="CoinDesk">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:221]:
```suggestion
								<img src="assets/client-logos/CoinDesk_logo.png" alt="CoinDesk" loading="lazy">
```

File: wwwroot/index.html [hunk 12]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://coinpick.today/wasabi-fulfills-security-expectations-to-become-the-most-secure-bitcoin-wallet/"
+								target="_blank"> <img src="assets/client-logos/coinpick.png" alt="coinpick"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:225]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/coinpick.png" alt="coinpick" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 13]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide" href="https://cryptochan.org/stream/id/1532515705/" target="_blank"> <img
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:226]:
```suggestion
							<a class="slide" href="https://cryptochan.org/stream/id/1532515705/" target="_blank" rel="external nofollow"> <img
```

File: wwwroot/index.html [hunk 14]
```diff
@@ -0,0 +1,534 @@
...
+									src="assets/client-logos/cryptochan.png" alt="cryptochan">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:227]:
```suggestion
									src="assets/client-logos/cryptochan.png" alt="cryptochan" loading="lazy">
```

File: wwwroot/index.html [hunk 15]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide" href="https://cryptoinfobase.com/wasabi-wallet-btc-anonymity/"
+								target="_blank"> <img src="assets/client-logos/cryptoinfobase.png" alt="cryptoinfobase">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/cryptoinfobase.png" alt="cryptoinfobase" loading="lazy">
```

File: wwwroot/index.html [hunk 16]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide" href="https://www.cryptotraders.eu/what-is-wasabi-wallet/" target="_blank">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:233]:
```suggestion
							<a class="slide" href="https://www.cryptotraders.eu/what-is-wasabi-wallet/" target="_blank" rel="external nofollow">
```

File: wwwroot/index.html [hunk 17]
```diff
@@ -0,0 +1,534 @@
...
+								<img src="assets/client-logos/cryptotraders.png" alt="cryptotraders"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:234]:
```suggestion
								<img src="assets/client-logos/cryptotraders.png" alt="cryptotraders" loading="lazy"> </a>
```

**Review by RiccardoMasutti — CHANGES_REQUESTED** [review]

File: wwwroot/index.html [hunk 18]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide"
+								href="https://finance.yahoo.com/news/bitcoin-privacy-takes-another-step-203038327.html"
+								target="_blank"> <img src="assets/client-logos/Yahoo_Finance_Logo.png"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:237]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/Yahoo_Finance_Logo.png"
```

File: wwwroot/index.html [hunk 19]
```diff
@@ -0,0 +1,534 @@
...
+									alt="Yahoo_Finance"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html]:
```suggestion
									alt="Yahoo_Finance" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 20]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide"
+								href="https://www.reddit.com/r/Bitcoin/comments/8wk63k/wasabi_privacy_focused_bitcoin_wallet_for_desktop/"
+								target="_blank"> <img src="assets/client-logos/reddit.png" alt="reddit"></a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:242]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/reddit.png" alt="reddit" loading="lazy"></a>
```

File: wwwroot/index.html [hunk 21]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide"
+								href="https://www.ccn.com/bitcoin-privacy-takes-another-step-forward-with-wasabi-wallet-launch/"
+								target="_blank"> <img src="assets/client-logos/CCN-Logo.png" alt="CCN-Logo"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:245]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/CCN-Logo.png" alt="CCN-Logo" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 22]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide"
+								href="https://medium.com/@nopara73/wasabi-privacy-focused-bitcoin-wallet-for-desktop-3962d567045a"
+								target="_blank"> <img src="assets/client-logos/medium-logo.jpg" alt="medium-logo">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:248]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/medium-logo.jpg" alt="medium-logo" loading="lazy">
```

File: wwwroot/index.html [hunk 23]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://cryptonewmedia.press/2018/07/07/wasabi-wallet-signals-a-new-level-of-bitcoin-privacy/?utm_source=dlvr.it&amp;utm_medium=twitter"
+								target="_blank"> <img src="assets/client-logos/crypto-new-media.jpg"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:252]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/crypto-new-media.jpg"
```

File: wwwroot/index.html [hunk 24]
```diff
@@ -0,0 +1,534 @@
...
+									alt="crypto-new-media"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html]:
```suggestion
									alt="crypto-new-media" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 25]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide"
+								href="https://bitcoinexchangeguide.com/wasabi-bitcoin-wallet-privacy-focused-crypto-wallet-launched-for-desktop/"
+								target="_blank"> <img src="assets/client-logos/Bitcoin-Exchange-Guide.png"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:257]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/Bitcoin-Exchange-Guide.png"
```

File: wwwroot/index.html [hunk 26]
```diff
@@ -0,0 +1,534 @@
...
+									alt="Bitcoin-Exchange-Guide">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:258]:
```suggestion
									alt="Bitcoin-Exchange-Guide" loading="lazy">
```

File: wwwroot/index.html [hunk 27]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide" href="https://steemit.com/bitcoin/@sames/wasabi-wallet-bitcoin-privacy"
+								target="_blank"> <img src="assets/client-logos/steemit.png" alt="steemit"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:261]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/steemit.png" alt="steemit" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 28]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide"
+								href="https://www.tokendaily.co/p/wasabi-privacy-focused-bitcoin-wallet-for-desktop"
+								target="_blank"> <img src="assets/client-logos/cryptowiki.png" alt="cryptowiki">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:264]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/cryptowiki.png" alt="cryptowiki" loading="lazy">
```

File: wwwroot/index.html [hunk 29]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://cryptowiki.ru/article/wasabi-wallet-obeshaet-sdelat-bitkoin-tranzakcii-deistvitelno-privatnymi.html"
+								target="_blank"> <img src="assets/client-logos/nulltx.png" alt="nulltx"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:268]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/nulltx.png" alt="nulltx" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 30]
```diff
@@ -0,0 +1,534 @@
...
+							<a class="slide" href="https://nulltx.com/what-is-wasabi-wallet/" target="_blank"> <img
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:269]:
```suggestion
							<a class="slide" href="https://nulltx.com/what-is-wasabi-wallet/" target="_blank" rel="external nofollow"> <img
```

File: wwwroot/index.html [hunk 31]
```diff
@@ -0,0 +1,534 @@
...
+									src="assets/client-logos/Coin-News-Telegraph-logo.png" alt="Coin-News-Telegraph">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:270]:
```suggestion
									src="assets/client-logos/Coin-News-Telegraph-logo.png" alt="Coin-News-Telegraph" loading="lazy">
```

File: wwwroot/index.html [hunk 32]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://cryptovest.com/features/privacy-focused-bitcoin-wallet-wasabi-launches-beta-version-on-uasf-anniversary/"
+								target="_blank"> <img src="assets/client-logos/Bitcoin-Insight-Logo.png"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:274]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/Bitcoin-Insight-Logo.png"
```

File: wwwroot/index.html [hunk 33]
```diff
@@ -0,0 +1,534 @@
...
+									alt="Bitcoin-Insight">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:275]:
```suggestion
									alt="Bitcoin-Insight" loading="lazy">
```

File: wwwroot/index.html [hunk 34]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="http://www.bitcoin-insight.com/privacy-focused-bitcoin-wallet-wasabi-launches-beta-version-on-uasf-anniversary/"
+								target="_blank"> <img src="assets/client-logos/btc_manager.png" alt="btc_manager">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:279]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/btc_manager.png" alt="btc_manager" loading="lazy">
```

File: wwwroot/index.html [hunk 35]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://btcmanager.com/making-bitcoin-transactions-untraceable-with-the-wasabi-wallet/"
+								target="_blank"> <img src="assets/client-logos/Insidebitcoins.png" alt="Insidebitcoins">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:283]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/Insidebitcoins.png" alt="Insidebitcoins" loading="lazy">
```

File: wwwroot/index.html [hunk 36]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://insidebitcoins.com/news/making-bitcoin-transactions-untraceable-with-the-wasabi-wallet/175864"
+								target="_blank"> <img src="assets/client-logos/coinstaker.jpg" alt="coinstaker">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:287]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/coinstaker.jpg" alt="coinstaker" loading="lazy">
```

File: wwwroot/index.html [hunk 37]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://bitcoinmagazine.com/articles/bitcoin-privacycoin-tech-making-bitcoin-more-private/"
+								target="_blank"> <img src="assets/client-logos/Coinrussia.png" alt="Coinrussia">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:291]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/Coinrussia.png" alt="Coinrussia" loading="lazy">
```

File: wwwroot/index.html [hunk 38]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://coinrussia.ru/blockchain/the-release-of-the-beta-version-of-wasabi-wallet/"
+								target="_blank"> <img src="assets/client-logos/blockpublisher_logo.png"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:295]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/blockpublisher_logo.png"
```

File: wwwroot/index.html [hunk 39]
```diff
@@ -0,0 +1,534 @@
...
+									alt="blockpublisher">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:296]:
```suggestion
									alt="blockpublisher" loading="lazy">
```

File: wwwroot/index.html [hunk 40]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://blockpublisher.com/wasabi-fulfills-security-expectations-to-become-the-most-secure-bitcoin-wallet/"
+								target="_blank"> <img src="assets/client-logos/digitalbitcoinlogo.png"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:300]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/digitalbitcoinlogo.png"
```

File: wwwroot/index.html [hunk 41]
```diff
@@ -0,0 +1,534 @@
...
+									alt="digitalbitcoin">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:301]:
```suggestion
									alt="digitalbitcoin" loading="lazy">
```

File: wwwroot/index.html [hunk 42]
```diff
@@ -0,0 +1,534 @@
...
+							</a>
+							<a class="slide"
+								href="https://www.nasdaq.com/article/bitcoin-as-a-privacycoin-this-tech-is-making-bitcoin-more-private-cm1016978"
+								target="_blank"> <img src="assets/client-logos/Bitcoin_Kurier.png" alt="Bitcoin_Kurier">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:305]:
```suggestion
								target="_blank" rel="external nofollow"> <img src="assets/client-logos/Bitcoin_Kurier.png" alt="Bitcoin_Kurier" loading="lazy">
```

File: wwwroot/index.html [hunk 43]
```diff
@@ -0,0 +1,534 @@
...
+					<div class="col-12">
+						<h1 class="text-center mb-5 card-title underline-contact-title">Contact</h1>
+					</div>
+				</div>
+
+				<div class="row justify-content-center">
+					<div class="col-lg-3 col-md-6 col-12">
+						<div class="card ">
+							<div class="card-body">
+								<img class="contact-picture" src="assets/contact/thumb-1.jpg">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:331]:
```suggestion
								<img class="contact-picture" src="assets/contact/thumb-1.jpg" loading="lazy">
```

File: wwwroot/index.html [hunk 44]
```diff
@@ -0,0 +1,534 @@
...
+								<h4 class="card-title">GERGELY HAJDU</h4>
+								<hr>
+								<p class="card-text">Co-Founder, CEO</p>
+								<div class="contact-logos">
+									<a href="mailto:gergely.hajdu@zksnacks.com"><img src="assets/envelope-solid.svg"
+											alt="email">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:337]:
```suggestion
											alt="email" loading="lazy">
```

File: wwwroot/index.html [hunk 45]
```diff
@@ -0,0 +1,534 @@
...
+									</a>
+									<a href="https://www.linkedin.com/in/gergelyhajdu" target="_blank"><img class="ml-1"
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:309]:
```suggestion
									<a href="https://www.linkedin.com/in/gergelyhajdu" target="_blank" rel="external nofollow"><img class="ml-1"
```

File: wwwroot/index.html [hunk 46]
```diff
@@ -0,0 +1,534 @@
...
+											src="assets/linkedin-in-brands.svg" alt="linkedin"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:340]:
```suggestion
											src="assets/linkedin-in-brands.svg" alt="linkedin" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 47]
```diff
@@ -0,0 +1,534 @@
...
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="col-lg-3 col-md-6 col-12">
+						<div class="card ">
+							<div class="card-body">
+								<img class="contact-picture" src="assets/contact/thumb-2.jpg">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:348]:
```suggestion
								<img class="contact-picture" src="assets/contact/thumb-2.jpg" loading="lazy">
```

File: wwwroot/index.html [hunk 48]
```diff
@@ -0,0 +1,534 @@
...
+								<h4 class="card-title">BALINT HARMAT</h4>
+								<hr>
+								<p class="card-text">Co-Founder, CEO</p>
+								<div class="contact-logos">
+									<a href="mailto:balint.harmat@zksnacks.com"> <img src="assets/envelope-solid.svg"
+											alt="email">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:354]:
```suggestion
											alt="email" loading="lazy">
```

File: wwwroot/index.html [hunk 49]
```diff
@@ -0,0 +1,534 @@
...
+									</a>
+									<a href="https://www.linkedin.com/in/balint-harmat-0a673634/" target="_blank"> <img
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:326]:
```suggestion
									<a href="https://www.linkedin.com/in/balint-harmat-0a673634/" target="_blank" rel="external nofollow"> <img
```

File: wwwroot/index.html [hunk 50]
```diff
@@ -0,0 +1,534 @@
...
+											class="ml-1" src="assets/linkedin-in-brands.svg" alt="linkedin"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:357]:
```suggestion
											class="ml-1" src="assets/linkedin-in-brands.svg" alt="linkedin" loading="lazy"> </a>
```

**Review by RiccardoMasutti — CHANGES_REQUESTED** [review]

File: wwwroot/index.html [hunk 51]
```diff
@@ -0,0 +1,534 @@
...
+			<div class="container">
+				<div class="row topdiv">
+					<div class="col-12">
+						<div class="fadethisdiv">
+							<div class="card-body">
+								<h2 class="card-title underline-title mb-4">Mission</h2>
+								<p class="card-text text-center">
+									zkSNACKs was established to research the possibilities for Bitcoin fungibility
+									enhancements and provide different products ensuring the desired privacy.
+									<a class="bitcoin-anonymous" href="https://bitcoin.org/en/protect-your-privacy">Many
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:88]:
```suggestion
									<a class="bitcoin-anonymous" href="https://bitcoin.org/en/protect-your-privacy" target="_blank">Many
```

File: wwwroot/index.html [hunk 52]
```diff
@@ -0,0 +1,534 @@
...
+									<a href="https://www.linkedin.com/in/balint-harmat-0a673634/" target="_blank"> <img
+											class="ml-1" src="assets/linkedin-in-brands.svg" alt="linkedin"> </a>
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="col-lg-3 col-md-6 col-12">
+						<div class="card ">
+							<div class="card-body">
+								<img class="contact-picture" src="assets/contact/1112.jpg">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:365]:
```suggestion
								<img class="contact-picture" src="assets/contact/1112.jpg" loading="lazy">
```

File: wwwroot/index.html [hunk 53]
```diff
@@ -0,0 +1,534 @@
...
+								<h4 class="card-title">ADAM FICSOR </h4>
+								<hr>
+								<p class="card-text">Co-Founder, CTO</p>
+								<div class="contact-logos">
+									<a href="mailto:adam.ficsor@zksnacks.com"> <img src="assets/envelope-solid.svg">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:370]:
```suggestion
									<a href="mailto:adam.ficsor@zksnacks.com"> <img src="assets/envelope-solid.svg" loading="lazy">
```

File: wwwroot/index.html [hunk 54]
```diff
@@ -0,0 +1,534 @@
...
+									</a>
+									<a href="https://github.com/nopara73/" target="_blank"> <img class="ml-1"
+											src="assets/github-brands.svg" alt="github"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:373]:
```suggestion
											src="assets/github-brands.svg" alt="github" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 55]
```diff
@@ -0,0 +1,534 @@
...
+									<a href="https://twitter.com/nopara73" target="_blank"> <img class="ml-1"
+											src="assets/twitter-brands.svg" alt="twitter"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:375]:
```suggestion
											src="assets/twitter-brands.svg" alt="twitter" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 56]
```diff
@@ -0,0 +1,534 @@
...
+									<a href="https://medium.com/@nopara73" target="_blank"> <img class="ml-1"
+											src="assets/medium-brands.svg" alt="medium"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:377]:
```suggestion
											src="assets/medium-brands.svg" alt="medium" loading="lazy"> </a>
```

File: wwwroot/index.html [hunk 57]
```diff
@@ -0,0 +1,534 @@
...
+								</div>
+							</div>
+						</div>
+					</div>
+					<div class="col-lg-3 col-md-6 col-12">
+						<div class="card ">
+							<div class="card-body">
+								<img class="contact-picture" src="assets/contact/unnamed.png">
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:385]:
```suggestion
								<img class="contact-picture" src="assets/contact/unnamed.png" loading="lazy">
```

File: wwwroot/index.html [hunk 58]
```diff
@@ -0,0 +1,534 @@
...
+								<h4 class="card-title">LUCAS ONTIVERO</h4>
+								<hr>
+								<p class="card-text">Lead Engineer</p>
+								<div class="contact-logos">
+									<a href="https://github.com/lontivero" target="_blank"> <img
+											src="assets/github-brands.svg" alt="github"> </a>
```
Comment (RiccardoMasutti) [inline wwwroot/index.html:391]:
```suggestion
											src="assets/github-brands.svg" alt="github" loading="lazy"> </a>
```

**Review by RiccardoMasutti — CHANGES_REQUESTED** [review]

**Comment by RiccardoMasutti** [comment]
@molnard can you apply those updates please? Just small SEO tweaks

**Comment by molnard** [comment]
The source is handled by a Hungarian web dev, he will do the changes. @hgergoil what is the status of this?

**Comment by hgergoil** [comment]
End of this week Balazs will finish it.

**Comment by molnard** [comment]
Main menu is too transparent 

image

**Comment by molnard** [comment]
Merging, @Rmartin1104 pls create PR with @RiccardoMasutti's changes.

**Pull request merged**